PWA container fails to start because pnpm is unable to install @api-platform/admin

**API Platform version(s) affected**:3.0.8

**Description**  
On a fresh instance of the Api-platform distribution, the PWA container fails to start because pnpm can't install the admin package

**How to reproduce**  
1. Download and extract the release .tar.gz
2. run `docker compose build --pull --no-cache`
3. run `docker compose up`
4. pwa container throws an error and shuts down

**Additional Context**  
PWA container shows the following log
```bash
2022-12-28 23:34:19 Lockfile is up to date, resolution step is skipped
2022-12-28 23:34:19 Progress: resolved 1, reused 0, downloaded 0, added 0
2022-12-28 23:34:19 .                                        | +458 ++++++++++++++++++++++++++++++++
2022-12-28 23:34:19 undefined
2022-12-28 23:34:19  ERR_PNPM_NO_OFFLINE_TARBALL  A package is missing from the store but cannot download it in offline mode. The missing package may be downloaded from https://registry.npmjs.org/@api-platform/admin/-/admin-3.4.3.tgz.
2022-12-28 23:34:19 
2022-12-28 23:34:19 > pwa@0.1.0 dev /srv/app
2022-12-28 23:34:19 > next dev
2022-12-28 23:34:19 
2022-12-28 23:34:19 sh: next: not found
2022-12-28 23:34:19  ELIFECYCLE  Command failed.
```

--- a/pwa/Dockerfile
+++ b/pwa/Dockerfile
@@ -9,7 +9,7 @@ WORKDIR /srv/app
 
 RUN corepack enable && \
 	corepack prepare --activate pnpm@latest && \
-	echo "store-dir=/.pnpm-store\n" > /etc/npmrc
+	pnpm config -g set store-dir /.pnpm-store
 
 # Next.js collects completely anonymous telemetry data about general usage.
 # Learn more here: https://nextjs.org/telemetry

--- a/pwa/Dockerfile
+++ b/pwa/Dockerfile
@@ -9,7 +9,7 @@ WORKDIR /srv/app
 
 RUN corepack enable && \
 	corepack prepare --activate pnpm@latest && \
-    pnpm config set store-dir /.pnpm-store
+	echo "store-dir=/.pnpm-store\n" > /etc/npmrc
 
 # Next.js collects completely anonymous telemetry data about general usage.
 # Learn more here: https://nextjs.org/telemetry

